packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v2-2-x/deploy-cc-from-golang-source.test.ts

```diff
@@ -46,248 +42,216 @@ import { DEFAULT_FABRIC_2_AIO_IMAGE_NAME } from "@hyperledger/cactus-test-toolin
+    const assetTransferGoPath = path.join(contractDir, "./assetTransfer.go");
+    const assetTransferGoBuf = await fs.readFile(assetTransferGoPath);
+    const assetTransferGo = {
+      body: assetTransferGoBuf.toString("base64"),
+      filename: `${contractName}.go`,
+    };
+    const goModPath = path.join(contractDir, "./go.mod");
+    const goModBuf = await fs.readFile(goModPath);
+    const goMod = {
+      body: goModBuf.toString("base64"),
+      filename: "go.mod",
+    };
+
+    const goSumPath = path.join(contractDir, "./go.sum");
+    const goSumBuf = await fs.readFile(goSumPath);
+    const goSum = {
+      body: goSumBuf.toString("base64"),
+      filename: "go.sum",
+    };
+    const res = await apiClient.deployContractV1({
+      channelId,
+      ccVersion: "1.0.0",
+      // constructorArgs: { Args: ["john", "99"] },
+      sourceFiles: [assetTransferGo, smartContractGo, goMod, goSum],
+      ccName: contractName,
+      targetOrganizations: [
+        FABRIC_25_LTS_FABRIC_SAMPLES_ENV_INFO_ORG_1,
+        FABRIC_25_LTS_FABRIC_SAMPLES_ENV_INFO_ORG_2,
+      ],
+      caFile:
+        FABRIC_25_LTS_FABRIC_SAMPLES_ENV_INFO_ORG_1.ORDERER_TLS_ROOTCERT_FILE,
+      ccLabel: "basic-asset-transfer-2",
+      ccLang: ChainCodeProgrammingLanguage.Golang,
+      ccSequence: 1,
+      orderer: "orderer.example.com:7050",
+      ordererTLSHostnameOverride: "orderer.example.com",
+      connTimeout: 60,
+    });
+    const { packageIds, lifecycle, success } = res.data;
+    expect(res.status).toBe(200);
+    expect(success).toBeTruthy();
+    const {
+      approveForMyOrgList,
+      installList,
+      queryInstalledList,
+      commit,
+      packaging,
+      queryCommitted,
+    } = lifecycle;
+    expect(packageIds).toBeTruthy();
+    expect(Array.isArray(packageIds)).toBe(true);
+    expect(approveForMyOrgList).toBeTruthy();
+    expect(Array.isArray(approveForMyOrgList)).toBe(true);
+    expect(installList).toBeTruthy();
+    expect(Array.isArray(installList)).toBe(true);
+    expect(queryInstalledList).toBeTruthy();
+    expect(Array.isArray(queryInstalledList)).toBe(true);
+    expect(commit).toBeTruthy();
+    expect(packaging).toBeTruthy();
+    expect(queryCommitted).toBeTruthy();
+
+    const assetId = uuidv4();
+    const assetOwner = uuidv4();
+    // CreateAsset(id string, color string, size int, owner string, appraisedValue int)
+    const createRes = await apiClient.runTransactionV1({
+      contractName,
+      channelName,
+      params: [assetId, "Green", "19", assetOwner, "9999"],
+      methodName: "CreateAsset",
+      invocationType: FabricContractInvocationType.Send,
+      signingCredential: {
+        keychainId,
+        keychainRef: keychainEntryKey,
+      },
+    });
+    expect(createRes).toBeTruthy();
+    expect(createRes.status).toBeGreaterThan(199);
+    expect(createRes.status).toBeLessThan(300);
+
+    console.log(
+      `BassicAssetTransfer.Create(): ${JSON.stringify(createRes.data)}`,
+    );
```

Comment:
Please use logger instead of `console.log`

Reply:
Same applies to other files as well (entire PR)

Reply:
@outSH I double  checked and there is no more console.log in the diff - resolving the thread so we can merge